script.js

```diff
@@ -1,4 +1,114 @@
+//Project 2: Beat That (Base+Comfortable)
+
+var rollDice = "ROLL DICE";
+var chooseSequence = "CHOOSE SEQUENCE";
+var gameMode = rollDice;
+var player1;
+var player2;
```

Comment:
Naming for these variables can be more specific, just by looking at the variable name I'm not sure if this is referring to player's name, score, final result etc.